File: src/test/samples/sample.cs
// #region FirstRegion
int x = 42;
// #endregion

// #region SecondRegion
class MyClass {
    //  #region InnerRegion  
    void MyMethod() {}
    //#endregion   ends  InnerRegion 

    // #region
    void MyMethod2() {}
    //    #endregion  
}
// #endregion

File: src/test/samples/sample.rb
#region FirstRegion
x = 42
#endregion

#   region SecondRegion
class MyClass
    #  region    InnerRegion
    def my_method
    end
    # endregion   ends InnerRegion

    #region
    def my_method2
    end
    #   endregion
end
# endregion
